tests/vcs/git/GitCommitBearTest.py

```diff
@@ -87,62 +87,73 @@ def test_git_failure(self):
         git_error = self.msg_queue.get().message
         self.assertEqual(git_error[:4], 'git:')
 
-        self.assert_no_msgs()
```

Comment:
Why remove this test ?, why not handle the case when the folder isn't a git repository in the VCSCommitBear itself ?

Reply:
I have just removed the `assert_no_msgs()` check because it does get a message now which is `HEAD commit information`